AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1542,13 +1542,13 @@ petri.com#?#.content_widget:has(> div.widget_content > div.hs-form-insert)
 fort-russ.com#?#.single-container > div.better-studio-shortcode:has(> div.bs-push-noti-inner)
 gatevidyalay.com#?#aside > section.widget-sidebar:has(> div.textwidget > section > form[id^="subscribe-"])
 it-actual.ru##.widget-area > aside.widget > div[class] form[method="post"]
-thebell.io#?#.sidebar-item:has(> .connect)
-thebell.io#?#.sidebar-item:has(> div[id*="ad-sidebar-"])
-thebell.io#?#.sidebar-item:has(> div:contains(weekly newsletter))
+thebell.io##.sidebar
 thebell.io##.follow > .follow__title
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow-form-line
-thebell.io##.follow > .follow-form > .follow-form-cell >.follow__example
-thebell.io#?#.follow > .follow-form > .follow-form-cell:has(> .button)
+thebell.io##.follow > .follow__subtitle
+thebell.io##.follow > .follow-form
```

Comment:
```
thebell.io##.follow > .follow__subtitle
thebell.io##.follow > .follow-form
```
what about `thebell.io##whatabout`?:) This rule will hide 
![image](https://user-images.githubusercontent.com/8361299/77061489-fa3c4280-69e2-11ea-8cf4-405295dcb776.png)

Reply:
It's too wide, we need to keep whatabout block, it's a tl;dr version of an article.

Reply:
ok